.circleci/config.yml

```diff
@@ -382,76 +370,98 @@ jobs:
 
   working_directory: ~/repo
 
-  build:
+  install:
     executor: node
     environment:
       # required to prevent ENOMEM errors
       LERNA_ARGS: --concurrency 1
     steps:
       - checkout
-      - detect_root_change
-      - cache_checksum
       - restore_cache:
-          name: Restore Yarn Package Cache
-          keys:
-            - yarn-packages-{{ checksum "yarn.lock" }}
-      - restore_node_modules_cache
+            name: Restore Yarn Package Cache
+            keys:
+              - yarn-packages-{{ checksum "yarn.lock" }}
       - install_node_dependencies
-      - save_cache:
-          name: Save Yarn Package Cache
-          key: yarn-packages-{{ checksum "yarn.lock" }}
-          paths:
-            - ~/.cache/yarn
-      - save_cache:
-          key: yarn-dependencies-{{ .Branch }}-{{ checksum "~/.tmp/checksumfiles/yarn.lock" }}
-          paths:
-            - node_modules
       - run: # Run PR Checks
-          name: Run PR Checks
-          command: yarn danger ci
+         name: Run PR Checks
+         command: yarn danger ci
+      - persist_to_workspace:
```

Comment:
Rather than caching, we use `persist_to_workspace` to copy everything in the repo over to the next job to reduce duplication of steps